docs/getting-started/styling-with-twind.md

```diff
@@ -107,7 +107,34 @@ tw({
 
 ## The apply function
 
-To be documented...
+Use {@link twind.apply | apply} to inline any existing utility classes into a single CSS class.
+
+This is useful when you find a common utility pattern in your HTML that you'd like to extract to a new component.
```

Comment:
How does this work with more complex hierarchies? If possible, it would be great to have a more complicated example.

Reply:
Good idea. I try to include a complex example.

Reply:
I will merge this now and add documentation to the new site.